apps/src/blockly/addons/cdoFieldBitmap.ts

```diff
@@ -37,4 +38,11 @@ export class CdoFieldBitmap extends FieldBitmap {
     const bitmap = JSON.parse(fieldElement.textContent || '[]');
     this.setValue(bitmap);
   }
+
+  setValue(newValue: number[][], fireChangeEvent = true) {
+    super.setValue(newValue, fireChangeEvent);
+    if (newValue?.length) {
```

Comment:
(can be a follow-up): Can you add a comment explaining why we need to do this and what the calculation is doing?

Reply:
Yes! Will do a fast follow tomorrow